sdk/v1/client.go

```diff
@@ -233,6 +235,45 @@ func (cli *Client) GetJob(ctx context.Context, reqheader client.Headers, jobID s
 	return &respHeaders, &job, nil
 }
 
+func (cli *Client) GetJobs(ctx context.Context, reqheader client.Headers, options client.Options) (*client.RespHeaders, *models.Jobs, error) {
+	if reqheader.ServiceAuthToken == "" {
+		reqheader.ServiceAuthToken = cli.serviceToken
+	}
+
+	configValues, _ := config.Get()
+	validOffset, err := cli.ValidateOptions(options.Offset)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	validLimit, err := cli.ValidateOptions(options.Limit)
+	if err != nil {
+		validLimit = strconv.Itoa(configValues.DefaultMaxLimit)
```

Comment:
Sorry Bhupendra but on second thoughts I think we need to return the error here too:
```suggestion
		return nil, nil, err
```